src/main/java/com/summitsync/api/model/BaseTemplate.java

```diff
@@ -0,0 +1,23 @@
+package com.summitsync.api.model;
+
+import jakarta.persistence.*;
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+import java.util.List;
+
+@MappedSuperclass
+@NoArgsConstructor
+@AllArgsConstructor
+@Data
+public class BaseTemplate {
+    @Id
+    @GeneratedValue
+    private long btId;
+    private String acronym;
+    private int numberOfDates;
+    private String description;
+    @ManyToMany
```

Comment:
Ich glaube wir sollten hier (und in allen anderen Klasse auch) den FetchType explizit angeben. Wahrscheinlich lazy?